ui/admin/app/components/roles/role/actions/index.hbs

```diff
@@ -3,38 +3,21 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-{{!TODO Refactor page to use hds dropdown in ICU-13879}}
-<Rose::Dropdown
-  @text={{t 'actions.manage'}}
-  @dropdownRight={{true}}
-  as |dropdown|
->
+<Hds::Dropdown as |dd|>
+  <dd.ToggleButton @text= {{t 'actions.manage'}} @color ='secondary'/>
   {{#if (can 'addPrincipals role' @model)}}
-    <dropdown.link
-      @route='scopes.scope.roles.role.add-principals'
-      data-test-manage-dropdown-principals
-    >
-      {{t 'resources.role.principal.actions.add-principals'}}
-    </dropdown.link>
+    <dd.Interactive
+      @text={{t 'resources.role.principal.actions.add-principals'}}
+      @route = 'scopes.scope.roles.role.add-principals'
+    />
   {{/if}}
-
-  {{#if (can 'setGrantScopes role' @model)}}
-    <dropdown.link
-      @route='scopes.scope.roles.role.manage-scopes'
-      data-test-manage-dropdown-scopes
-    >
-      {{t 'resources.role.scope.actions.manage-scopes'}}
-    </dropdown.link>
```

Comment:
This link to the manage-scopes routes might have accidentally been deleted but it is still needed.